common/src/main/java/org/opensearch/ml/common/FunctionName.java

```diff
@@ -7,6 +7,7 @@
 
 public enum FunctionName {
     LINEAR_REGRESSION,
+    LOGISTIC_REGRESSION,
```

Comment:
We'd better put the new algorithm to last position. It may cause issue for a cluster with mixed version.

Reply:
Will do in the revision. These two are both regressions so listed them together in the first place.